buildSrc/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+    compile 'com.android.tools.build:gradle:1.2.3'
```

Comment:
Gah good catch. local plugins are annoying with this since they can't look at the root project's declared extensions -____-

567ce40